From: Ole Streicher <olebole@debian.org>
Date: Thu, 14 Mar 2019 22:53:56 +0100
Subject: Disable numpy matrix deprecation warnings

Numpy matrix deprecation warnings are disabled by python-scipy 1.1.0-4,
so we shall make sure that the tests don't fail then.

Closes: #924606
---
 skimage/segmentation/tests/test_random_walker.py | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/skimage/segmentation/tests/test_random_walker.py b/skimage/segmentation/tests/test_random_walker.py
index 7a4c819..2ad153f 100644
--- a/skimage/segmentation/tests/test_random_walker.py
+++ b/skimage/segmentation/tests/test_random_walker.py
@@ -14,10 +14,7 @@ SCIPY_RANK_WARNING = r'numpy.linalg.matrix_rank|\A\Z'
 PYAMG_MISSING_WARNING = r'pyamg|\A\Z'
 PYAMG_OR_SCIPY_WARNING = SCIPY_RANK_WARNING + '|' + PYAMG_MISSING_WARNING
 
-if Version(scipy.__version__) < '1.3':
-    NUMPY_MATRIX_WARNING = 'matrix subclass'
-else:
-    NUMPY_MATRIX_WARNING = None
+NUMPY_MATRIX_WARNING = None
 
 
 def make_2d_syntheticdata(lx, ly=None):
